http://bugs.gentoo.org/101773

===================================================================
RCS file: /home/wine/wine/programs/winelauncher.in,v
retrieving revision 1.4
retrieving revision 1.5
diff -u -p -r1.4 -r1.5
--- wine/programs/winelauncher.in	2004/06/21 23:56:15	1.4
+++ wine/programs/winelauncher.in	2005/08/10 10:51:50	1.5
@@ -59,8 +59,8 @@ type xmessage >/dev/null 2>/dev/null
 if [ $? -ne 0 ] ; then
     # xmessage not found; make sure the user notices this error
     # (GUI users wouldn't even notice if we printed the text on console !)
-    MSGFILE=/tmp/WINE_CANNOT_FIND_XMESSAGE
-    cat > $MSGFILE << EOF
+    MSGFILE=`mktemp "/tmp/wine.xmessage.XXXXXX"`
+    cat > $MSGFILE <<EOF
 Warning:
     The Wine launcher is unable to find the xmessage program,
     which it needs to properly notify you of Wine execution status
@@ -87,6 +87,7 @@ EOF
 
     # ok, we really give up now, this system is hosed ;-)
     cat $MSGFILE
+    rm $MSGFILE
 else
     XMESSAGE="xmessage $COLOR"
 fi
